cargo-pgx/src/command/install.rs

```diff
@@ -400,6 +400,9 @@ pub(crate) fn get_version(manifest_path: impl AsRef<Path>) -> eyre::Result<Strin
                     .wrap_err("Couldn't parse manifest")?;
 
                 let version = manifest.package.ok_or(eyre!("no `[package]` section found"))?.version;
+                let Inheritable::Set(version) = version else {
```

Comment:
I should make sure the manifest is parsed with https://docs.rs/cargo_toml/latest/cargo_toml/struct.Manifest.html#method.complete_from_path, so that this case can't happen.